packages/wrangler/src/deploy/deploy.ts

```diff
@@ -11,8 +11,16 @@ import {
 	printBundleSize,
 	printOffendingDependencies,
 } from "../deployment-bundle/bundle-reporter";
+import { getBundleType } from "../deployment-bundle/bundle-type";
 import { createWorkerUploadForm } from "../deployment-bundle/create-worker-upload-form";
-import traverseModuleGraph from "../deployment-bundle/traverse-module-graph";
+import {
+	findAdditionalModules as doFindAdditionalModules,
```

Comment:
Why the rename?

Reply:
Because we also have a `findAdditionalModules` variable further down the file; and eslint wasn't happy.

Reply:
I only see https://github.com/cloudflare/workers-sdk/blob/3612d62cda34f4bc7a56c56ccf7690b11bd10016/packages/wrangler/src/deploy/deploy.ts#L473 which eslint shouldn't be complaining about since it's not a variable being shadowed

Reply:
LOL! An additional refactor appears to have removed the destructuring. So you're right!